### PR TITLE
Improved Input Parsing and Unit Test 

### PR DESCRIPTION
**Pull request by shac170**
- Made boundary conditions not case sensitive
- Changed boundary condition error message to tell user the acceptable options
- Made plane and cylinder inputs use space, underscore or tab
- Made unit test for the above features

### CODE REVIEW

File: mcdc/input_.py [hunk 1]
```diff
@@ -187,16 +187,17 @@ def surface(type_, **kw):
     # Boundary condition
     bc = kw.get('bc')
     if bc is not None:
+        bc=bc.lower()
```
Comment (jpmorgan98) [inline mcdc/input_.py:190]: spaces around equals, PEP8

File: test/test_unit/test_input_.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+
```
Comment (jpmorgan98) [inline test/test_unit/test_input_.py:1]: same pep8 issue

**Review by jpmorgan98 — CHANGES_REQUESTED** [review]
pep8 issues

**Review by jpmorgan98 — APPROVED** [review]
everything looks good!

**Review by goodman17c — APPROVED** [review]
Good addition!
